lib/project_types/script/layers/infrastructure/script_service.rb

```diff
@@ -83,6 +83,8 @@ def set_app_script(
               valid_types: e["message"],
               filename: script_config.filename,
             )
+          elsif (e = user_errors.find { |err| err["tag"] == "input_query_validation_error" })
```

Comment:
Is it possible that there could be multiple errors and we want to display all of them? Similar to what we do with [configuration errors](https://github.com/Shopify/shopify-cli/blob/58a4576e0ff77d6972491b7a7254eab14f4bb279/lib/project_types/script/layers/infrastructure/script_service.rb#L55-L59)?

Reply:
Done